cluster/gce/util.sh

```diff
@@ -325,6 +325,19 @@ function detect-master () {
   echo "Using master: $KUBE_MASTER (external IP: $KUBE_MASTER_IP)"
 }
 
+# Reads kube-env metadata from master
+#
+# Assumed vars:
+#   KUBE_MASTER
+#   PROJECT
+#   ZONE
+function get-master-env() {
+  # TODO(zmerlynn): Make this more reliable with retries.
```

Comment:
nit: How did zmerlynn's comment get in here?  Is this code cut 'n pasted from somewhere else, not reflected in this PR?  Just curious, mainly.